tests/playwright-test/playwright.expect.true.spec.ts

```diff
@@ -176,6 +176,39 @@ test('should support toBeEditable, toBeEnabled, toBeDisabled, toBeEmpty', async
   expect(output).toContain('expect(locator).toBeEnabled({ timeout: 500 }');
 });
 
+test('should support toBeDisabled,toBeChecked,toBeHidden w/ value', async ({ runInlineTest }) => {
```

Comment:
```suggestion
test('should support toBeDisabled,toBeChecked,toBeHidden', async ({ runInlineTest }) => {
```

Reply:
That's what people had in the bug actually: `disabled="disabled"`